src/inference/include/openvino/runtime/intel_gpu/properties.hpp

```diff
@@ -124,6 +124,8 @@ static constexpr Property<int64_t> available_device_mem{"AVAILABLE_DEVICE_MEM_SI
  */
 static constexpr Property<bool> enable_sdpa_optimization{"GPU_ENABLE_SDPA_OPTIMIZATION"};
 
+static constexpr Property<bool> disable_flashattnv2_optimization{"GPU_DISABLE_FLASHATTNV2_OPTIMIZATION"};
```

Comment:
should it be internal GPU option? 
or you think it can be useful for end users as well?

Reply:
@ceciliapeng2011, agree with Ilya
Can we set it internally based on some model's pattern and don't expose it to users?

Reply:
@sshlyapn  Would you please give an example of an "internal" config option? Does it mean users cannot either config it through compile_model(), or with environment variable?

Reply:
@ceciliapeng2011 
https://github.com/openvinotoolkit/openvino/blob/637fc24286f0f3c96aa9fd11afad4b7f14eb29be/src/plugins/intel_gpu/include/intel_gpu/runtime/internal_properties.hpp#L116-L132

>  Does it mean users cannot either config it through compile_model(), or with environment variable?

yes, they cannot

Reply:
done